Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md

```diff
@@ -6,6 +6,17 @@ versionFrom: 7.0.0
 
 All new projects on Umbraco Cloud are automatically protected by Umbraco Latch. This means, that the default Umbraco Cloud URL for your project as well as any new hostnames you add will be assigned a TLS certificate automatically.
 
+:::note
+Certificates last 3 months and are automatically renewed, as long as the prerequisites below are met.
+:::
+
+## Prerequisites
+In order for Latch to work and the certificate to be applied the root of the site that the domain is pointing to cannot have the status of 404 whether it's a custom 404 or one caused by an error.
```

Comment:
I'm not entirely sure I understand this sentence.
Is the point that, a domain cannot have a 404 status?

```suggestion
In order for Latch to work and the certificate to be applied to the root of the site that the domain is pointing to cannot have the status of 404 whether it's a custom 404 or one caused by an error.
```